automation/const.go

```diff
@@ -0,0 +1,10 @@
+package automation
+
+// Paths that will be used for future tests
+const (
+	DbSecretPath  = "../config/db-secret.yaml"
+	NdbSecretPath = "../config/ndb-secret.yaml"
+	DbPath        = "./database.yaml"
+	AppPodPath    = "../config/pod.yaml"
```

Comment:
Would you need to update these paths, If you move the db-engine specific yaml to another folder like @manavrajvanshi has suggested above?